frontend/packages/dev-console/src/components/topology/TopologyPage.tsx

```diff
@@ -112,6 +125,7 @@ const TopologyPage: React.FC<Props> = ({
                 to={`/topology/${namespace ? `ns/${namespace}` : 'all-namespaces'}${
                   showListView ? '' : '/list'
                 }`}
+                onClick={() => setTopologyActiveView(showListView ? 'topology' : 'list')}
```

Comment:
Why not use `useEffect` so that we save the value even if the user enters the URL manually instead of only when they click the button?
```
React.useEffect(() => setTopologyActiveView(showListView ? 'topology' : 'list'), [showListView]);
```

Reply:
Ah, I missed this scenario. I will use ```useEffect``` to save the value.

Reply:
You can remove this now that you have switched to `useEffect`.

Reply:
Nm, you do need this because without it you cannot switch back to the graph view.
We may want to consider adding a new route `/topology/graph` which is for the graph view so that we have a direct URL that is linkable to send to others.